src/avfoundation.cs

```diff
@@ -254,6 +255,113 @@ interface AVMutableDateRangeMetadataGroup
 		AVMetadataItem[] Items { get; set; }
 	}
 	
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVDepthData
+	{
+		[Static]
+		[Export ("depthDataFromDictionaryRepresentation:error:")]
+		[return: NullAllowed]
+		AVDepthData Create (NSDictionary imageSourceAuxDataInfoDictionary, [NullAllowed] out NSError outError);
+
+		[Export ("depthDataByConvertingToDepthDataType:")]
+		AVDepthData Create (uint depthDataType);
+
+		[Export ("depthDataByApplyingExifOrientation:")]
+		AVDepthData Create (CGImagePropertyOrientation exifOrientation);
+
+		[Export ("depthDataByReplacingDepthDataMapWithPixelBuffer:error:")]
+		[return: NullAllowed]
+		AVDepthData CreateD (CVPixelBuffer pixelBuffer, [NullAllowed] out NSError outError);
+
+		[Export ("availableDepthDataTypes")]
+		NSNumber[] AvailableDepthDataTypes { get; }
+
+		[Export ("dictionaryRepresentationForAuxiliaryDataType:")]
+		[return: NullAllowed]
+		NSDictionary GetDictionaryRepresentation ([NullAllowed] out string outAuxDataType);
+
+		[Export ("depthDataType")]
+		uint DepthDataType { get; }
```

Comment:
Looks like there is a list of types, so we could probably do a  strong typed enum:

/*!
 @property depthDataType
 @abstract
    Specifies the pixel format type of this depth data object's internal map.
 
 @discussion
    One of kCVPixelFormatType_DisparityFloat16, kCVPixelFormatType_DisparityFloat32, kCVPixelFormatType_DepthFloat16, or kCVPixelFormatType_DepthFloat32.
 */